fsociety.py

```diff
@@ -466,7 +466,7 @@ def install(self):
     def run(self):
         clearScr()
         print(self.nmapLogo)
-        target = raw_input(self.targetPrompt)
+        target = raw_input(self.targetPrompt).split(' ')[0]
```

Comment:
Splitting the `hostname` or the `url`, which are `user-supplied` inputs doesn't prevent malicious attackers from injecting successfully other commands like `ls` or `shutdown`, which could provoke anyway a DOS issue or unexpected behaviors.

The `target` should be sanitized properly :smile:

Reply:
You can confirm what I'm saying just inputting as `target` the following `url`: `test.cominvalid&&ls` , and you'll receive in the `final output` also the `ls` command execution, which shows the file in the directory.

Video attached: 
![ezgif-2-85e2420b9482](https://user-images.githubusercontent.com/33063403/88428612-80cfb900-cdf5-11ea-83d3-5ff563598b25.gif)